src/Components/CreateImageWizard/steps/Details/index.tsx

```diff
@@ -24,13 +27,32 @@ const DetailsStep = () => {
   const dispatch = useAppDispatch();
   const blueprintName = useAppSelector(selectBlueprintName);
   const blueprintDescription = useAppSelector(selectBlueprintDescription);
+  const distribution = useAppSelector(selectDistribution);
+  const arch = useAppSelector(selectArchitecture);
+  const targetEnvironments = useAppSelector(selectImageTypes);
+
+  useEffect(() => {
+    if (!blueprintName) {
+      dispatch(changeBlueprintName(generateDefaultName()));
+    }
+  }, [dispatch, blueprintName]);
```

Comment:
Won't this generate a name when I type a name and then delete it?
I think that using empty dependency array is correct in this case.

Reply:
thanks